client/src/components/layouts/Default.js

```diff
@@ -145,13 +146,16 @@ class DefaultLayout extends Component {
       isSignedIn,
       removeFlashMessage,
       showFooter = true,
-      theme = 'default'
+      theme = 'default',
+      useTheme = true
     } = this.props;
     return (
       <Fragment>
         <Helmet
           bodyAttributes={{
-            class: `${theme === 'default' ? 'light-palette' : 'dark-palette'}`
+            class: useTheme
```

Comment:
Is this just intended to be until a dark theme version is created?

Reply:
Yes, the current servicebot css gets messed up with our darkmode styles.